TFT/src/User/API/Printing.c

```diff
@@ -374,6 +381,9 @@ void printRemoteStart(const char * filename)
 {
   infoHost.printing = true;  // always set (even if printing from onboard media)
 
+  if (MENU_IS(menuMarlinMode))  // do not process any printing info if Marlin Mode is active
```

Comment:
mmh, just because parseAck will receive and handle also "Not SD printing" and "SD printing byte" and so it will call setPrintPause() and setPrintResume() (that will access infoPrinting, infoFile.source etc..), it is probably better to let printRemoteStart() initialize those data structures and simply avoid to switch to the Printing menu (so it is enough to move your  `if (MENU_IS(menuMarlinMode))...` just before `infoMenu.cur = 1;`. There should also no need to apply any check/change in printEnd (or printComplete). We will simply hear a beep at the end of the print (or on print cancel). But that should be (for me) a good thing to have

Reply:
I would not recommend handling any print information at all apart from the status that a print is going on outside of the TFT. No print information should be processed while in Marlin Mode. 

>There should be also no need to apply any check/change in printEnd (or printComplete). We will simply hear a beep at the end of the print (or on print cancel). But that should be (for me) a good thing to have

This should be avoided because most of the hardware is required to be deinitialized for Marlin Mode to work properly like the buzzer and SD card (on some boards) is also used by Marlin. if the buzzer is not deinitialized by Touch mode, it will not work with Marlin Mode. This is also true for Encoder active signal to avoid encoder reading by Marlin mode when it should not.

Reply:
Yes that's also true. So if (in printRemoteStart()) we do not set infoPrinting, infoFile etc..., infoPrinting.printing will always remains set to false (e.g. by infoPrinting definition) so functions printEnd() will do nothing (it exits immediately) and there is no need to check if we are in Marlin Mode, setPrintResume will not set infoHost.printing to true (because infoPrinting.printing is set to false).